enterprise/server/remote_execution/containers/firecracker/BUILD

```diff
@@ -159,6 +161,59 @@ go_test(
         "//server/remote_cache/action_cache_server",
         "//server/remote_cache/byte_stream_server",
         "//server/remote_cache/content_addressable_storage_server",
+        "//server/remote_cache/digest",
+        "//server/testutil/testauth",
+        "//server/testutil/testcache",
+        "//server/testutil/testdigest",
+        "//server/testutil/testenv",
+        "//server/testutil/testfs",
+        "//server/util/disk",
+        "//server/util/fileresolver",
+        "//server/util/log",
+        "//server/util/networking",
+        "//server/util/status",
+        "//server/util/testing/flags",
+        "@com_github_stretchr_testify//assert",
+        "@com_github_stretchr_testify//require",
+        "@org_golang_google_genproto_googleapis_bytestream//:bytestream",
+        "@org_golang_x_sync//errgroup",
+    ],
+)
+
+go_test(
```

Comment:
it's kind of a pain having these separate test targets because Gazelle only manages the `firecracker_test` deps for us and we have to manage the other ones manually. I'm hoping to remove the `_blockio` target as soon as we launch the copy-on-write stuff.

Ideally we could split this into a separate test file like `firecracker_benchmark_test.go` and I think that should be enough for gazelle to manage it for us, not 100% sure though. (getTestEnv etc. could be moved into a separate testutil package)

Another option is to rename the test case to `BenchmarkFirecrackerSnapshotting` and then we can run it with `--test_filter=Benchmark --test_arg=-test.bench=. --test_arg=-test.benchtime=1x` - the downside though is that we can't run it automatically as part of our Benchmark action on CI

Reply:
Good point. To simplify, I just removed conditionally running this test in our regular test suite.

Reply:
This test seems pretty heavyweight (takes > 1min to run from looking at the logs) - I think we should keep it manually run for now

Reply:
If I split it into a separate test file, gazelle tries to add it as an additional `src` to the `firecracker_test` target, which won't let us set custom tags on it. To simplify, I just added a new flag to control whether this is run, and we can manually set it.